chucks-pick3-web/angular-app/src/app/services/form-validation.service/form-validation.service.ts

```diff
@@ -7,10 +7,14 @@ export class FormValidationService {
   constructor() {
   }
 
+/*ADD validateForm here!*/
+public validateForm() {
+return true;
```

Comment:
Be sure to have proper indentions.  Check all TS files for proper indentions.

Reply:
Formatted text accordingly.